src/core_plugins/kibana/server/lib/export/export_dashboards.js

```diff
@@ -0,0 +1,21 @@
+import _ from 'lodash';
+import collectDashboards from './collect_dashboards';
+import { SavedObjectsClient } from '../../../../../server/saved_objects';
+
+export const deps = { collectDashboards };
+
+export default function exportDashboards(req) {
+  const ids = _.flatten([req.payload || req.query.dashboard]);
```

Comment:
Doesn't look like the payload is validated by the `dashboard/export` route, why accept it as the dashboard id?